CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project are documented in this file.
 
+## Unreleased
+
+ - Added the Model Adapters `FilterModel` and `MapModel`.
+ - CRUD example rewritten to use Model Adapters.
```

Comment:
we don't usually add changes to the examples in the ChangeLog. I think the changelog should be more geared towards user of the slint library.